src/items.c

```diff
@@ -2831,3 +2859,249 @@ void HideSpawnPoints()
+		{
+			if (self->ct != ctPlayer)
+			{
+				G_sprint(self, 2, "Error: spectators cannot change timings\n");
+
+				return;
+			}
+		}
+
+		if (!k_matchLess && match_in_progress)
+		{
+			G_sprint(self, 2, "Error: match currently in progress\n");
+
+			return;
+		}
+
+		trap_CmdArgv(2, buf, sizeof(buf));
+		if (!streq(buf, "default"))
+		{
+			seconds = strtof(buf, &end);
+			if (end == buf || seconds <= 0)
+			{
+				G_sprint(self, 2, "Error: invalid duration\n");
+
+				return;
+			}
+		}
+	}
+
+	trap_CmdArgv(1, buf, sizeof(buf));
+	if (streq(buf, "health"))
+	{
+		if (argc == 2)
+		{
+			PrintTiming(ITEM_CLASS_HEALTH, "Health respawn time: ");
+		}
+		else
+		{
+			SetTiming(ITEM_CLASS_HEALTH, seconds, 5, 1500);
+		}
+	}
+	else if (streq(buf, "ammo"))
+	{
+		if (argc == 2)
+		{
+			PrintTiming(ITEM_CLASS_AMMO, "Ammo respawn time: ");
+		}
+		else
+		{
+			SetTiming(ITEM_CLASS_AMMO, seconds, 5, 1500);
+		}
+	}
+	else if (streq(buf, "armor"))
+	{
+		if (argc == 2)
+		{
+			PrintTiming(ITEM_CLASS_ARMOR, "Armor respawn time: ");
+		}
+		else
+		{
+			SetTiming(ITEM_CLASS_ARMOR, seconds, 5, 1500);
+		}
+	}
+	else if (streq(buf, "weapon"))
+	{
+		if (argc == 2)
+		{
+			PrintTiming(ITEM_CLASS_WEAPON, "Weapon respawn time: ");
+		}
+		else
+		{
+			SetTiming(ITEM_CLASS_WEAPON, seconds, 5, 1500);
+		}
+	}
+	else if (streq(buf, "mega"))
+	{
+		if (argc == 2)
+		{
+			PrintTiming(ITEM_CLASS_MEGA, "Mega respawn time: ");
+		}
+		else
+		{
+			SetTiming(ITEM_CLASS_MEGA, seconds, 5, 1500);
+		}
+	}
+	else if (streq(buf, "quad"))
+	{
+		if (argc == 2)
+		{
+			PrintTiming(ITEM_CLASS_QUAD, "Quad respawn time: ");
+		}
+		else
+		{
+			SetTiming(ITEM_CLASS_QUAD, seconds, 5, 1500);
+		}
+	}
+	else if (streq(buf, "eyes"))
+	{
+		if (argc == 2)
+		{
+			PrintTiming(ITEM_CLASS_EYES, "Eyes respawn time: ");
+		}
+		else
+		{
+			SetTiming(ITEM_CLASS_EYES, seconds, 5, 1500);
+		}
+	}
+	else if (streq(buf, "pent"))
+	{
+		if (argc == 2)
+		{
+			PrintTiming(ITEM_CLASS_PENT, "Pent respawn time: ");
+		}
+		else
+		{
+			SetTiming(ITEM_CLASS_PENT, seconds, 5, 1500);
+		}
+	}
+	else if (streq(buf, "all"))
+	{
+		if (argc == 2)
+		{
+			G_sprint(self, 2, "Respawn times:\n");
+			PrintTiming(ITEM_CLASS_HEALTH, " Health: ");
+			PrintTiming(ITEM_CLASS_AMMO,   " Ammo:   ");
+			PrintTiming(ITEM_CLASS_ARMOR,  " Armor:  ");
+			PrintTiming(ITEM_CLASS_WEAPON, " Weapon: ");
+			PrintTiming(ITEM_CLASS_MEGA,   " Mega:   ");
+			PrintTiming(ITEM_CLASS_QUAD,   " Quad:   ");
+			PrintTiming(ITEM_CLASS_EYES,   " Eyes:   ");
+			PrintTiming(ITEM_CLASS_PENT,   " Pent:   ");
+		}
+		else if (seconds == 0)
+		{
+			ResetTimings();
+		}
+		else
+		{
+			G_sprint(self, 2, "Error: invalid duration\n");
+		}
+	}
+	else
+	{
+		G_sprint(self, 2, "Error: invalid item class\n");
+	}
+}
+
+qbool CustomTimingsActive()
+{
+	int i;
+
+	for (i = 0; i < NUM_ITEM_CLASSES; i++)
+	{
+		if (respawn_times[i] != default_respawn_times[deathmatch][i])
+		{
+			return 1;
+		}
+	}
+	return 0;
```

Comment:
a newline character is missing before the return statement

Reply:
Easy fix